dolphinscheduler-e2e/dolphinscheduler-e2e-core/src/main/java/org/apache/dolphinscheduler/e2e/core/DolphinSchedulerExtension.java

```diff
@@ -59,12 +59,13 @@
 import com.google.common.net.HostAndPort;
 
 import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.utility.DockerImageName;
 
 @Slf4j
-final class DolphinSchedulerExtension
-    implements BeforeAllCallback, AfterAllCallback,
-    BeforeEachCallback {
-    private final boolean LOCAL_MODE = Objects.equals(System.getProperty("local"), "true");
+final class DolphinSchedulerExtension implements BeforeAllCallback, AfterAllCallback, BeforeEachCallback {
+    private final boolean LOCAL_MODE = Objects.equals(System.getenv("local"), "true");
+
+    private final boolean M1_CHIP_FLAG = Objects.equals(System.getenv("m1_chip"), "true");
```

Comment:
We are using system properties, not environment variables

Reply:
I'll fix it.